open-metadata-implementation/access-services/digital-service/digital-service-api/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/events/DigitalServiceEventHeader.java

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.digitalservice.events;
+
+import com.fasterxml.jackson.annotation.*;
```

Comment:
Generally it's preferred to be explicit about imports rather than using *

Reply:
addressed

Reply:
I don't see a change here in c22e3dfcf2b03fecfa9544e33258987eb91d4359

Reply:
added

Reply:
Hi - I pulled the latest code to check, and there's no change there, so perhaps the update didn't get in?

Reply:
fixed